technologies/app/saagie-code-search/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1
+type: APP
+id: saagie-code-search
+label: Saagie Code Search
+baseline: "Saagie Code Search"
+description: "Saagie-Code-Search: Custom app based on Flask that Saagie Code Search is a web application that enables the search for Python functions, either within a default codebase or GitHub repositories."
+available: true
+icon: datascience
+backgroundColor: "#253c5b"
+customFlags: []
+contexts:
+  - id: saagie-code-search-0.1
+    label: For Saagie V1
```

Comment:
For the label, can you set 
`label: For Saagie V2`

In order to have the same thing between this app and the app Saagie HF ModelServer TextCLF